v2.0/create-sequence.md

```diff
@@ -58,13 +58,38 @@ In this example, we create a sequence with default settings.
 > CREATE SEQUENCE customer_seq;
 ~~~
 
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW CREATE SEQUENCE customer_seq;
+~~~
+~~~
++--------------+------------------------------------------------------------------------------------------+
+|   Sequence   |                                      CreateSequence                                      |
++--------------+------------------------------------------------------------------------------------------+
+| customer_seq | CREATE SEQUENCE customer_seq MINVALUE 1 MAXVALUE 9223372036854775807 INCREMENT 1 START 1 |
++--------------+------------------------------------------------------------------------------------------+
+~~~
+
 ### Create a Sequence with User-Defined Settings
 
-In this example, we create a sequence that starts at 1000 and descends in increments of 2.
+In this example, we create a sequence that starts at -1 and descends in increments of 2.
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE SEQUENCE desc_customer_list START -1 INCREMENT -2;
+~~~
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> CREATE SEQUENCE desc_customer_list START 1000 INCREMENT -2;
+> SHOW CREATE SEQUENCE desc_customer_list;
+~~~
+
+~~~
++--------------------+----------------------------------------------------------------------------------------------------+
+|      Sequence      |                                           CreateSequence                                           |
++--------------------+----------------------------------------------------------------------------------------------------+
+| desc_customer_list | CREATE SEQUENCE desc_customer_list MINVALUE -9223372036854775808 MAXVALUE -1 INCREMENT -2 START -1 |
++--------------------+----------------------------------------------------------------------------------------------------+
 ~~~
 
 ### Create a Table with a Sequence
```

Comment:
I like that this example demonstrates using `nextval()`. You may want to add another example or two demonstrating other sequence functions. Perhaps @vilterp has ideas.

Reply:
Added another example to ALTER SEQUENCE - "Set the Next Value of a Sequence"